src/lib/frontend/cnf.ml

```diff
@@ -84,7 +84,7 @@ let rec make_term up_qv quant_basename t =
       end
 
     | TTprefix ((Sy.Op Sy.Minus) as s, n) ->
-      let t1 = if ty == Ty.Tint then E.int "0" else E.real "0"  in
+      let t1 = if ty == Ty.Tint then E.Ints.of_int 0 else E.Reals.of_int 0 in
```

Comment:
```suggestion
      let t1 = if ty == Ty.Tint then E.Ints.of_Z Z.zero else E.Reals.of_Q Q.zero in
```
Can be applied everywhere else

Reply:
Is it better?

Reply:
Marginally, if at all. I guess it's more a matter of taste than performance.